solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -486,23 +491,20 @@ contract RandomBeacon is Ownable {
             "Operator is already registered"
         );
 
-        sortitionPool.insertOperator(operator);
+        sortitionPool.insertOperator(
+            operator,
+            staking.eligibleStake(msg.sender, address(this))
+        );
     }
 
     /// @notice Updates the sortition pool status of the caller.
     function updateOperatorStatus() external {
         sortitionPool.updateOperatorStatus(
```

Comment:
Did we remove `sortitionPool.getOperatorID()` function from SP?

Reply:
It is still there but `updateOperatorStatus` accepts `address` now.